resources/.ktlint_editorconfig

```diff
@@ -4,3 +4,11 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 ktlint_standard_multiline-expression-wrapping = disabled
 ktlint_standard_string-template-indent = disabled
+ktlint_standard_chain-method-continuation = disabled
+ktlint_standard_class-signature = disabled
+ktlint_standard_trailing-comma-on-call-site = disabled
+ktlint_standard_function-signature = disabled
+ktlint_standard_trailing-comma-on-declaration-site = disabled
+ktlint_standard_if-else-wrapping = disabled
```

Comment:
I would prefer requiring the use of {}'s for multiline if-statements, personally. It seems maybe only a little weird to disallow them for single line if-statements.

Reply:
I support this - although inline ifs are a neat kotlin feature my java-brain finds it hard to read these inline if then else statements!